ckan/tests/cli/test_cli.py

```diff
@@ -110,3 +111,26 @@ def test_ckan_config_loader_parse_two_files():
     assert conf[u'global_conf'][u'__file__'] == filename
     assert conf[u'global_conf'][u'here'] == tpl_dir
     assert conf[u'global_conf'][u'debug'] == u'false'
+
+
+def test_chain_loading():
+    """Load chains of config files via `use = config:...`.
+    """
+    filename = os.path.join(
+        os.path.dirname(__file__), u'data', u'test-one.ini')
+    conf = CKANConfigLoader(filename).get_config()
+    assert conf[u'__file__'] == filename
+    assert conf[u'key1'] == u'one'
+    assert conf[u'key2'] == u'two'
+    assert conf[u'key3'] == u'three'
+
+
+def test_recursive_loading():
+    """ Make sure we still remember main config file.
+
+    If there are circular dependencies, make sure the user knows about it.
+    """
+    filename = os.path.join(
+        os.path.dirname(__file__), u'data', u'test-one-recursive.ini')
```

Comment:
Wrong name or missing file here

Reply:
Yeah, we have all *.ini files ignored by git and I forgot about it